src/calendar-heatmap.component.js

```diff
@@ -158,34 +158,31 @@ export class CalendarHeatmap extends Component {
       'year'
     );
 
+    const getSummary = (date) => {
+      // Create 'summary' dictionary: Record<string, {name: string; value: number}>
+      let summaryDictionary = this.props.data.reduce((summary, d) => {
+        if (moment(d.date).year() === date.year()) {
```

Comment:
Considering this line, see if `.filter().reduce()` should be done instead in line 163